esp/platform/espcfg.cpp

```diff
@@ -263,34 +263,36 @@ CEspConfig::CEspConfig(IProperties* inputs, IPropertyTree* envpt, IPropertyTree*
                 ptree = &pt_iter->query();
                 if (ptree)
                 {
-                    binding_cfg *bcfg = new binding_cfg;
-                    
+                    OwnedPtr<binding_cfg> bcfg(new binding_cfg);
```

Comment:
the if clause performed more actions before this change, please confirm the logic change doesn't introduce any side effects other then the stated port 0 binding change.